pallets/compliance-manager/src/lib.rs

```diff
@@ -703,7 +789,18 @@ impl<T: Trait> Module<T> {
         primary_issuance_agent: Option<IdentityId>,
     ) -> AssetComplianceResult {
         let asset_compliance = Self::asset_compliance(ticker);
+        // To know whether the given ticker has the implicit requirements active or not.
+        // if `yes` then it returns the `Some(ImplicitRequirementResult)` for the `from_did_opt` and `to_did_opt`.
+        // if `no` then it returns None.
+        let implicit_result = Self::get_implicit_condition_result(
+            ticker,
+            from_did_opt,
+            to_did_opt,
+            primary_issuance_agent,
+        );
         let mut asset_compliance_with_results = AssetComplianceResult::from(asset_compliance);
+        // `implicit_requirements_result` by default is `None` here but we assign them here as per the above results.
+        asset_compliance_with_results.implicit_requirements_result = implicit_result;
```

Comment:
You can directly assign the output of `get_implicit_condition_result` to `asset_compliance_with_results.implicit_requirements_result`.